.github/workflows/dropshots-screenshot-test.yml

```
@@ -0,0 +1,655 @@
+      - name: Extract prebuilt AVD for API ${{ matrix.api-level }}
+        run: |
+          if [ -f "prebuilt-avd/avd-android-${{ matrix.api-level }}.tar.gz" ]; then
+            echo "Extracting prebuilt AVD for API ${{ matrix.api-level }}..."
+            tar xzf prebuilt-avd/avd-android-${{ matrix.api-level }}.tar.gz -C ~/.android/
+            echo "Prebuilt AVD extracted successfully"
+          else
+            echo "Prebuilt AVD not found, will create new AVD"
+          fi
+
+      - name: Setup Android SDK (if needed)
+        if: hashFiles('prebuilt-avd/avd-android-${{ matrix.api-level }}.tar.gz') == ''
```

actionlint (1.7.7)

if: condition "hashFiles('prebuilt-avd/avd-android-${{ matrix.api-level }}.tar.gz') == ''" is always evaluated to true because extra characters are around ${{ }}

(if-cond)